coreneuron/io/global_vars.cpp

```diff
@@ -55,22 +55,31 @@ void set_globals(const char* path, bool cli_global_seed, int cli_global_seed_val
         const char* name;
         int size;
         double* val = nullptr;
-        for (void* p = nullptr;
-             (p = (*nrn2core_get_global_dbl_item_)(p, name, size, val)) != nullptr;) {
-            N2V::iterator it = n2v->find(name);
-            if (it != n2v->end()) {
-                if (size == 0) {
-                    nrn_assert(it->second.first == 0);
-                    *(it->second.second) = val[0];
-                } else {
-                    nrn_assert(it->second.first == (size_t) size);
-                    double* pval = it->second.second;
-                    for (int i = 0; i < size; ++i) {
-                        pval[i] = val[i];
+        for (void* p = nullptr;;) {
+            p = (*nrn2core_get_global_dbl_item_)(p, name, size, val);
```

Comment:
`nitpick`: just looks bit better to my eyes 😃 
```suggestion
        while(1) {
            void* p = (*nrn2core_get_global_dbl_item_)(p, name, size, val);
```